notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4447,6 +4447,54 @@ with great power comes great responsibility [Ben shared exactly this aphorism wi
 journalism [Peter garnered some income by selling Spider-Man photos to the Daily Bugle tabloid newspaper.]
 
 
+movie: Thunderbirds (2004)
+==========================
+
+:: Title
+Thunderbirds
+
+:: Description
+Thunderbirds is a 2004 science fiction action-adventure film directed by
+Jonathan Frakes, based on the 1960s TV series Thunderbirds created by Gerry
+and Sylvia Anderson.
+
+Synopsis: A criminal mastermind traps the leader of a secret international
+rescue team together with four of his sons onboard a space station and
+proceeds to commit heists that the rescue team will be blamed for, prompting
+the father's youngest son and his friends to stop him.
+
+:: Date
+2004-07-20
+
+:: References
+https://en.wikipedia.org/wiki/Thunderbirds_(2004_film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+childhood aspirations [The teenager Alan was determined to follow in the footsteps of his four older brothers and become a Thunderbird pilot.]
+human aspiration [The teenager Alan was determined to follow in the footsteps of his four older brothers and become a Thunderbird pilot.]
```

Comment:
Desire for glory and fame?

Reply:
A better movie would have explore one or the other of those things. But we're talking 1/5 here...